core/frontend-devtools/package.json

```diff
@@ -39,6 +39,7 @@
     "@itwin/core-common": "workspace:*",
     "@itwin/core-frontend": "workspace:*",
     "@itwin/core-geometry": "workspace:*",
+    "@itwin/frontend-tiles": "workspace:*",
```

Comment:
Prefer to put the new tool in display-test-app and not introduce a dependency here on frontend-tiles.